pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -210,6 +210,13 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 		)
 	}
 
+	sanityExecute := func() {
+		watchtesting.SanityExecute(controller, []cache.Store{
+			controller.vmiIndexer, controller.vmStore, controller.podIndexer, controller.pvcIndexer,
+			controller.dataVolumeIndexer, controller.cdiStore, controller.cdiConfigStore,
+		}, Default)
```

Comment:
Any reason why storageClass and storageProfile are not checked?

Reply:
Oversight.

Reply:
Done